tools-local/setuptools/dotnet-deb-tool/tool/templates/debian/control

```diff
@@ -13,7 +13,7 @@ Homepage: {HOMEPAGE}
 
 Package: {PACKAGE_NAME}
 Architecture: {ARCH}
-Depends: ${{shlibs:Depends}}, ${{misc:Depends}}{DEPENDENT_PACKAGES}
+Depends: {DEPENDENT_PACKAGES}
```

Comment:
What does this change do?

Reply:
This enables building runtime packages (host,hostfxr and runtime ) without exhibiting system dependencies by default as we are moving away all system dependencies to runtime-deps package and consolidating all the system dependencies statically in distro specific runtime-deps config file.

Reply:
I assume you've tested installing these new installers on all our targeted platforms and it still works.  I'm not familiar enough with the way debhelper works to know if this is the right change or not - so I trust you've done the appropriate research here.

Reply:
Yes, I have tested installing all runtime packages on all targeted platforms. @wli3 is testing SDK with the private runtime packages to get another level of test coverage.